exchange/orders.php

```diff
@@ -184,9 +184,10 @@ function wc1c_replace_document_services($order, $document_services) {
   }
 
   if (!$shipping_methods) {
-    $shipping = WC()?->shipping;
-    $shipping?->load_shipping_methods();
-    $shipping_methods = $shipping?->get_shipping_methods();
```

Comment:
This is just a syntax sugar witch breakes PHP backward compatibility.

Reply:
Thanks, that's from Rector